src/lib/uri/uri.c

```diff
@@ -20,6 +126,7 @@ uri_destroy(struct uri *uri)
 	free(uri->path);
 	free(uri->query);
 	free(uri->fragment);
+	TRASH(uri);
```

Comment:
Nit: this belongs in the previous patch.

Reply:
fixed